spring-security-modules/spring-security-web-rest/src/main/java/com/baeldung/errorhandling/CustomRestExceptionHandler.java

```diff
@@ -45,8 +46,8 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(final MethodArgume
         return handleExceptionInternal(ex, apiError, headers, apiError.getStatus(), request);
     }
 
-    @Override
-    protected ResponseEntity<Object> handleBindException(final BindException ex, final HttpHeaders headers, final HttpStatus status, final WebRequest request) {
+    /*@Override
```

Comment:
Do we know why we have this commented method? Maybe it is mentioned in the article? If not needed maybe we can remove.

Reply:
it is not mentioned in the article I have removed this from code